### PR TITLE
Setup endpoints for database CRUD 

### PR DESCRIPTION
**Pull request by nullxDEADBEEF**
The idea is to add an admin layer to our application in which classes, students, etc. can be added through various forms

### CODE REVIEW

**Comment by nullxDEADBEEF** [comment]
Should probably split the routes into separate files

**Comment by AndreasPB** [comment]
Looking noooice so far

File: backend/app/db/crud.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+
+def create_studentclass(db: Session, studentclass: StudentClass):
+    db_studentclass = StudentClass(
+            id=studentclass.id,
+            name=studentclass.name,
+            number_of_students=studentclass.number_of_students,
+            courses=studentclass.courses,
+            lecturers=studentclass.lecturers
+    )
```
Comment (AndreasPB) [inline backend/app/db/crud.py]: Do unpacking pls
```suggestion
    db_studentclass = StudentClass(**studentclass)
```

Comment (nullxDEADBEEF) [reply backend/app/db/crud.py]: They are now unpacked in the newest commit :D

**Review by AndreasPB — APPROVED** [review]
We should make a follow up issue with the missing `get_all_x()` and the unimplemented `update_x_by_id()`, But other than that LGTM

**Comment by AndreasPB** [comment]
#27

**Pull request merged**